### PR TITLE
Refactor: Cleanup playground file

### PR DESCRIPTION
**Pull request by manthanguptaa**

### CODE REVIEW

File: phi/playground/operator.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from typing import List, Optional
+
+from phi.agent.agent import Agent, Tool, Toolkit, Function
+from phi.utils.log import logger
```
Comment (ashpreetbedi) [inline phi/playground/operator.py:4]: update import ordering

```
from phi.agent.session import AgentSession
from phi.utils.log import logger
```

File: phi/playground/playground.py [hunk 2]
```diff
@@ -1,59 +1,13 @@
...
-
-class GetAgentSessionsRequest(BaseModel):
-    agent_id: str
-    user_id: Optional[str] = None
-
-
-class GetAgentSessionsResponse(BaseModel):
-    session_id: Optional[str] = None
-    title: Optional[str] = None
+from .routes import create_playground_routes
```
Comment (ashpreetbedi) [inline phi/playground/playground.py:10]: not a fan of relative routes, lets use `phi.playground.routes`

File: phi/playground/playground.py [hunk 3]
```diff
@@ -1,59 +1,13 @@
...
+from typing import List, Optional
+from fastapi import FastAPI
 from fastapi.routing import APIRouter
-from fastapi.responses import StreamingResponse, JSONResponse
 
-from pydantic import BaseModel
-from phi.agent.agent import Agent, RunResponse, Tool, Toolkit, Function
+from phi.agent.agent import Agent
 from phi.playground.settings import PlaygroundSettings
 from phi.utils.log import logger
```
Comment (ashpreetbedi) [inline phi/playground/playground.py:7]: import ordering

File: phi/playground/playground.py (hunk 2 above)
Comment (ashpreetbedi) [reply phi/playground/playground.py:10]: more info: https://peps.python.org/pep-0008/#imports

File: phi/playground/routes.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
+from typing import List, Optional, Generator, Dict, cast, Union
+import base64
+from fastapi import APIRouter, HTTPException, UploadFile
+from fastapi.responses import StreamingResponse, JSONResponse
+
+from phi.agent.agent import Agent, RunResponse
+from phi.utils.log import logger
+from phi.agent.session import AgentSession
+from .operator import get_agent_by_id, get_session_title, format_tools
```
Comment (ashpreetbedi) [inline phi/playground/routes.py:9]: please use absolute imports

File: phi/playground/routes.py [hunk 5]
```diff
@@ -0,0 +1,133 @@
+from typing import List, Optional, Generator, Dict, cast, Union
+import base64
+from fastapi import APIRouter, HTTPException, UploadFile
+from fastapi.responses import StreamingResponse, JSONResponse
+
+from phi.agent.agent import Agent, RunResponse
+from phi.utils.log import logger
```
Comment (ashpreetbedi) [inline phi/playground/routes.py:7]: import ordering, phi.utils should be at the end

File: phi/playground/routes.py [hunk 6]
```diff
@@ -0,0 +1,133 @@
+from typing import List, Optional, Generator, Dict, cast, Union
+import base64
```
Comment (ashpreetbedi) [inline phi/playground/routes.py:2]: base64 should be at the top

File: phi/playground/routes.py [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+from fastapi import APIRouter, HTTPException, UploadFile
```
Comment (ashpreetbedi) [inline phi/playground/routes.py:3]: new line between std lib imports and 3rd party imports

**Review by ashpreetbedi — CHANGES_REQUESTED** [review]
Fantastic PR and mostly LGTM, just have a few comments regd imports

**Review by ysolanky — APPROVED** [review]

File: phi/playground/routes.py [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+
+from fastapi import APIRouter, HTTPException, UploadFile
+from fastapi.responses import StreamingResponse, JSONResponse
+
+from phi.agent.agent import Agent, RunResponse
+from phi.agent.session import AgentSession
+from phi.playground.operator import format_tools, get_agent_by_id, get_session_title
+from phi.utils.log import logger
+
+from .schemas import (
```
Comment (ysolanky) [inline phi/playground/routes.py:12]: @manthanguptaa here we should do `phi.playground.schemas`

Comment (manthanguptaa) [reply phi/playground/routes.py:12]: nice catch

**Pull request merged**